src/nvapi_d3d12.cpp

```diff
@@ -474,33 +475,41 @@ extern "C" {
         if (nvapiD3dInstance->IsUsingLfx() || !NvapiD3dLowLatencyDevice::SupportsLowLatency(pCommandQueue))
             return NoImplementation(n);
 
+        if (cqType == OUT_OF_BAND_IGNORE) {
+            if (!alreadyLoggedNotSupported)
+                log::info("NvAPI_D3D12_NotifyOutOfBandCommandQueue is called with OUT_OF_BAND_IGNORE, uncertain how to handle this type.");
+
+            // This would to an error anyway https://github.com/HansKristian-Work/vkd3d-proton/blob/master/libs/vkd3d/command_queue_vkd3d_ext.c#L67
+            return NotSupported(n, alreadyLoggedNotSupported);
```

Comment:
While logging this explicitly makes sense, I'd just let vkd3d-proton fail for us. This way we aren't blocking anything and should vkd3d-proton add support for this in the future, the message would become slightly misleading but not problematic.

Also, this would cause a funny result of us failing the call for `OUT_OF_BAND_IGNORE` but not for any other yet-undefined value, while dxvk-nvapi built against R550 headers would pass the value as-is and possibly work correctly with some future version of vkd3d-proton.

```suggestion
            if (!std::exchange(alreadyLoggedNotSupported, true))
                log::info("NvAPI_D3D12_NotifyOutOfBandCommandQueue called with OUT_OF_BAND_IGNORE.");
```

Reply:
My reasoning was to bail out fast with a clearer response code than just "Error". I'm quite certain that either me or your would catch up fast enough with vkd3d-proton if something changes there.  But I'm realizing that this change might be a small hurdle when somebody _is_ developing and testing the needed changes for vkd3d-proton. Will go for your suggestion.